src/plugins/discover/public/application/components/discover_legacy.tsx

```diff
@@ -0,0 +1,305 @@
+  timeRange,
+  topNavMenu,
+  updateQuery,
+  updateSavedQueryId,
+  vis,
+}: DiscoverLegacyProps) {
+  const [isSidebarClosed, setIsSidebarClosed] = useState(false);
+  const toMoment = function (datetime: string) {
+    if (!datetime) {
+      return '';
+    }
+    return moment(datetime).format(opts.config.get('dateFormat'));
+  };
+  const { TopNavMenu } = getServices().navigation.ui;
+  const { savedSearch, indexPatternList } = opts;
+  // @ts-ignore
+  const bucketInterval = vis?.data?.aggs?.aggs[1]?.buckets?.getInterval();
+
+  return (
+    <I18nProvider>
+      <div className="app-container" data-fetch-counter={fetchCounter}>
+        <h1 className="euiScreenReaderOnly">{savedSearch.title}</h1>
+        <TopNavMenu
+          appName="discover"
+          config={topNavMenu}
+          indexPatterns={[indexPattern]}
+          onQuerySubmit={updateQuery}
+          onSavedQueryIdChange={updateSavedQueryId}
+          query={state.query}
+          savedQueryId={state.savedQuery}
+          screenTitle={savedSearch.title}
+          showDatePicker={indexPattern.isTimeBased()}
+          showSaveQuery={showSaveQuery}
+          showSearchBar={true}
+          useDefaultBehaviors={true}
+        />
+        <main className="container-fluid">
+          <div className="row">
+            <div
+              className={`col-md-2 dscSidebar__container collapsible-sidebar ${
+                isSidebarClosed ? 'closed' : ''
+              }`}
+              id="discover-sidebar"
+              data-test-subj="discover-sidebar"
+            >
+              <div className="dscFieldChooser">
+                <DiscoverSidebar
+                  columns={state.columns || []}
+                  fieldCounts={fieldCounts}
+                  hits={rows}
+                  indexPatternList={indexPatternList}
+                  onAddField={addColumn}
+                  onAddFilter={onAddFilter}
+                  onRemoveField={onRemoveColumn}
+                  selectedIndexPattern={searchSource && searchSource.getField('index')}
+                  setIndexPattern={setIndexPattern}
+                />
+              </div>
+              <button
+                onClick={() => setIsSidebarClosed(!isSidebarClosed)}
+                data-test-subj="collapseSideBarButton"
+                aria-controls="discover-sidebar"
+                aria-expanded="true"
+                aria-label="Toggle sidebar"
+                className="kuiCollapseButton kbnCollapsibleSidebar__collapseButton"
+              >
+                <span
+                  className={`kuiIcon ${
+                    isSidebarClosed ? 'fa-chevron-circle-right' : 'fa-chevron-circle-left'
+                  }`}
+                />
+              </button>
+            </div>
+            <div className={`dscWrapper col-md-${isSidebarClosed ? '12' : '10'}`}>
+              {resultState === 'none' && (
+                <DiscoverNoResults
+                  timeFieldName={opts.timefield}
```

Comment:
So this was removed from 7.1 to 7.2, no failure variable in the scope. And since the shared error handling was in the meantime improved with PRs like https://github.com/elastic/kibana/pull/41649, I think this is save to remove